contrib/executor/jmeterd/pkg/slaves/client.go

```diff
@@ -51,8 +52,8 @@ func NewClient(execution testkube.Execution, envParams envs.Params, slavesEnvVar
 
 // creating slaves as per count provided in the SLAVES_CLOUNT env variable.
 // Default SLAVES_COUNT would be 1 if not provided in the env variables
-func (client *Client) CreateSlaves() (map[string]string, error) {
```

Comment:
We can modify comments for function

// CreateSlaves creates slaves as per count provided in the SLAVES_CLOUNT env variable.
// Default SLAVES_COUNT would be 1 if not provided in the env variables